02-create.md

```diff
@@ -54,6 +54,32 @@ Desktop/    pizza.cfg
 molecules/  solar.pdf
 ~~~
 
+> ## Good names for files and directories {.callout}
+>
+> Names of files and directories can make your life an hell
+> when working on the command line. Here we provide a few useful
+> tips for the names of your files from now on.
+>
+> 1. Don't use white spaces.
+>
+>    White spaces can make the name more meaningful
+>    but since it is used to break arguments on the command line
+>    is better to avoid them on name of files and directories.
+>    You can use `-` or `_` instead of white space.
+>
+> 2. Don't begin the name with `-`.
+>
+>    This will make some commands think that your file is an argument.
```

Comment:
Perhaps better: 'Some commands treat names starting with `-` as options'. It's simpler and also removes some confusion, because  filename are also arguments.